src/common/SIRF.py

```diff
@@ -202,6 +202,8 @@ def add(self, other, out=None):
         return z
     def axpby(self, a, b, y, out=None, **kwargs):
         '''
+        This function has been deprecated and will be removed in a future release. Please use `sapyb` instead.
```

Comment:
better to use https://python-deprecated.readthedocs.io/en/latest/tutorial.html?

Reply:
I'm happy to use this. But I'll need to add the `Python-Deprecated` to the requirements, could you point out where these live?

Reply:
should be there already
https://github.com/SyneRBI/SIRF/blob/master/requirements.txt#L7